.bumpversion.cfg

```
@@ -1,7 +1,7 @@
 [bumpversion]
 commit = True
 tag = True
-current_version = 0.2.1
+current_version = 0.3.0
```

```suggestion
current_version = 0.2.1
```
The version info will be auto updated via `bumpversion` CLI when bump new release.